src/Client.ts

```diff
@@ -71,33 +66,55 @@ export type XmtpEnv = keyof typeof ApiUrls
 /**
  * Network startup options
  */
-type NetworkOptions = {
-  // Allow for specifying different envs later
+export type NetworkOptions = {
+  /**
+   * Specify which XMTP environment to connect to. (default: `dev`)
+   */
   env: XmtpEnv
-  // apiUrl can be used to override the default URL for the env
+  /**
+   * apiUrl can be used to override the `env` flag and connect to a
+   * specific endpoint
+   */
   apiUrl: string | undefined
-  // app identifier included with client version header
+  /**
+   * app identifier, which will be included in headers of all API requests
```

Comment:
any recommended formats? Use their domain name?

Reply:
So far, all the apps are just using the app name (ie. `lenster/1.0.7` or `buttrfly/0.0.0`)

Reply:
Copied the full description from the README